examples/plot_simulate_evoked.py

```diff
@@ -18,32 +18,22 @@
 import mne_neuron
 from mne_neuron import simulate_dipole, Params, Network
 
-from neuron import h
-
-###############################################################################
-# Then we setup the directories and Neuron
 mne_neuron_root = op.join(op.dirname(mne_neuron.__file__), '..')
-h.load_file("stdrun.hoc")
 
 ###############################################################################
 # Then we read the parameters file
 params_fname = op.join(mne_neuron_root, 'param', 'default.json')
 params = Params(params_fname)
-print(params)
 
-###############################################################################
-# This is a lot of parameters! We can also filter the
-# parameters using unix-style wildcard characters
-print(params['L2Pyr_soma*'])
-
-###############################################################################
 # Now let's simulate the dipole
 net = Network(params)
 dpl = simulate_dipole(net)
 
 ###############################################################################
 # and then plot it
-import matplotlib.pyplot as plt
+import matplotlib
+matplotlib.use("TkAgg")
```

Comment:
I wouldn't hardcode the backend here. Users who need to use the TkAgg backend can set it in their matplotlibrc file.